releases/release-1.33/release-notes/release-notes-draft.json

```diff
@@ -0,0 +1,1701 @@
+      "node"
+    ],
+    "feature": true,
+    "duplicate": true
+  },
+  "128818": {
+    "commit": "438bc5d44e5b72cad99d6eb521d2eb3572c5b034",
+    "text": "Adds a /flagz endpoint for kube-scheduler endpoint",
+    "markdown": "Adds a /flagz endpoint for kube-scheduler endpoint ([#128818](https://github.com/kubernetes/kubernetes/pull/128818), [@yongruilin](https://github.com/yongruilin)) [SIG Architecture, Instrumentation, Scheduling and Testing]",
+    "documentation": [
+      {
+        "description": "[KEP]",
+        "url": "https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/4828-component-flagz/README.md",
+        "type": "KEP"
+      }
+    ],
+    "author": "yongruilin",
+    "author_url": "https://github.com/yongruilin",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/128818",
+    "pr_number": 128818,
+    "areas": [
+      "test"
+    ],
+    "kinds": [
+      "feature"
+    ],
+    "sigs": [
+      "architecture",
+      "instrumentation",
+      "scheduling",
+      "testing"
+    ],
+    "feature": true,
+    "duplicate": true
+  },
+  "128850": {
+    "commit": "beeb1d2b847ba302406f3a4b9390afe0853968b9",
+    "text": "Fix the issue where the named ports exposed by restartable init containers (a.k.a. sidecar containers) cannot be accessed using a Service.",
+    "markdown": "Fix the issue where the named ports exposed by restartable init containers (a.k.a. sidecar containers) cannot be accessed using a Service. ([#128850](https://github.com/kubernetes/kubernetes/pull/128850), [@toVersus](https://github.com/toVersus)) [SIG Network and Testing]",
+    "documentation": [
+      {
+        "description": "[KEP]",
+        "url": "https://github.com/kubernetes/enhancements/issues/753",
+        "type": "KEP"
+      }
+    ],
+    "author": "toVersus",
+    "author_url": "https://github.com/toVersus",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/128850",
+    "pr_number": 128850,
+    "areas": [
+      "test"
+    ],
+    "kinds": [
+      "bug"
+    ],
+    "sigs": [
+      "network",
+      "testing"
+    ],
+    "duplicate": true
+  },
+  "128856": {
+    "commit": "62e469abb2be372b45a6c095f7716a185cb5e827",
+    "text": "NOE",
+    "markdown": "NOE ([#128856](https://github.com/kubernetes/kubernetes/pull/128856), [@adrianmoisey](https://github.com/adrianmoisey)) [SIG Apps and Network]",
+    "author": "adrianmoisey",
+    "author_url": "https://github.com/adrianmoisey",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/128856",
+    "pr_number": 128856,
+    "kinds": [
+      "cleanup"
+    ],
+    "sigs": [
+      "apps",
+      "network"
+    ],
+    "duplicate": true
+  },
+  "128886": {
+    "commit": "30ef6110a180395ad55d1afa5dc60c90a761ae64",
+    "text": "kube-proxy nftables logs the failed transactions and the full table when using log level 4 or higher. Logging is rate limited to one entry every 24 hours to avoid performance issues.",
```

Comment:
```suggestion
    "text": "Implemented logging for failed transactions and the full table in `kube-proxy` with `nftables` when using log level 4 or higher. Logging is rate-limited to one entry every 24 hours to avoid performance issues.",
```